pep-0664.rst

```diff
@@ -0,0 +1,99 @@
+PEP: 664
+Title: Python 3.11 Release Schedule
+Version: $Revision$
+Last-Modified: $Date$
+Author: Pablo Galindo Salgado <pablogsal@python.org>
+Status: Draft
+Type: Informational
+Content-Type: text/x-rst
+Created: 12-Jul-2021
+Python-Version: 3.11
+
+
+Abstract
+========
+
+This document describes the development and release schedule for
+Python 3.11.  The schedule primarily concerns itself with PEP-sized
+items.
+
+.. Small features may be added up to the first beta
+   release.  Bugs may be fixed until the final release,
+   which is planned for end of October 2021.
+
+Release Manager and Crew
+========================
+
+- 3.11 Release Manager: Pablo Galindo Salgado
+- Windows installers: Steve Dower
+- Mac installers: Ned Deily
+- Documentation: Julien Palard
+
+
+Release Schedule
+================
+
+3.11.0 schedule
+---------------
+
+Note: the dates below use a 17-month development period that results
+in a 12-month release cadence between major versions, as defined by
+PEP 602.
+
+Actual:
+
+- 3.11 development begins: Monday, 2021-05-03
+
+Expected:
+
+- 3.11.0 alpha 1: Monday, 2021-10-04
+- 3.11.0 alpha 2: Tuesday, 2021-11-02
+- 3.11.0 alpha 3: Monday, 2021-12-06
+- 3.11.0 alpha 4: Monday, 2022-01-03
+- 3.11.0 alpha 5: Wednesday, 2022-02-02
+- 3.11.0 alpha 6: Monday, 2022-02-28
+- 3.11.0 alpha 7: Tuesday, 2022-04-05
+- 3.11.0 beta 1: Monday, 2022-05-02
```

Comment:
You might want to bump this a few days as May 2 is the first sprint day of PyCon US 2022:
- https://pycon.blogspot.com/2020/12/announcing-pycon-us-20222023.html

Reply:
```suggestion
- 3.11.0 beta 1: Monday, 2022-05-06
```

Reply:
It's not Monday anymore.